packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/aspnet/integrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="full">
+
+![Sequence diagram illustrating the password-optional sign-up use case.](/img/pwd-optional/pwd-optional-dotnet-sign-up-summary.png)
```

Comment:
Although somewhat of a nit, in step 4 there is an big empty white gap above the blue "Okta email enrollment flow" box.

Reply:
fixed